src/Utilities/constants.js

```diff
@@ -372,3 +372,25 @@ export const exportNotifications = (format) => ({
 });
 
 export const multiValueFilters = ['installed_evra', 'os'];
+
+export const environments = [
+    {
+        name: 'dev',
+        key: 'foo.redhat.com'
+    },
+    {
+        name: 'stage',
+        key: 'console.stage.redhat.com'
+    },
+    {
+        name: 'prod',
+        key: 'console.redhat.com'
+    }
+];
+
+export const featureFlags = [
+    {
+        name: 'patchSet',
+        urlKey: 'patch_set=enabled'
```

Comment:
This may not be best practice, but when I saw `patch_set=enabled`, I immediately thought, "what if I set `patch_set=disabled`"? Well, that would bypass the feature flag and show the content we are trying to hide. Maybe we do a check to see if it exists or is set to enabled. If it doesn't exist in the url, or it is set to disabled, then we block it.

Reply:
I will fix this also.It is definately confusion by me.  While impelementing, I did not think about disabling a feature as new features are not activated by default. They are only activated if the URL has associated urlKey or they are in stage environments.